flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/StreamingFileSink.java

```diff
@@ -205,33 +205,40 @@ private StreamingFileSink(
 			this.rollingPolicy = DefaultRollingPolicy.create().build();
 		}
 
+		RowFormatBuilder(
```

Comment:
This constructor can be `private`. Actually also the other constructor could be, as it is called by the enclosing class, but if it were to move the builder to a different file, then it would have a problem. So I would recommend to make the new constructor `private`.

Reply:
I would also call this constructor from the other one, just to have a single point of assignment of variables.